packages/stark-ui/src/modules/app-sidebar/services/app-sidebar.service.intf.ts

```diff
@@ -0,0 +1,39 @@
+import { Observable } from "rxjs";
+import { StarkAppSidebarOpenEvent } from "./app-sidebar-open-event.intf";
+
+/**
```

Comment:
Could you please provide a name and an injection token as we have in stark-core ?
```typescript
/**
 * The name of the service in case an injection is needed
 */
export const starkAppSidebarServiceName: string = "StarkAppSidebarService";
/**
 * The InjectionToken version of the service name
 */
export const STARK_APP_SIDEBAR_SERVICE: InjectionToken<StarkAppSidebarService> = new InjectionToken<StarkAppSidebarService>(starkAppSidebarServiceName);
```